### ISSUE TITLE
What does the 'text' parameter in the function 'generate_text_semantic' mean? It caused an error when cloning the voice

### ISSUE THREAD

**Issue by africa1207** [serp-ai/bark-with-voice-clone#16]
when i run this code `audio_array = generate_audio(text_prompt, history_prompt=voice_name, text_temp=0.7, waveform_temp=0.7)`,it prompt this error
![image](https://user-images.githubusercontent.com/21698755/234821737-89d54ffc-957f-4d51-829b-9b5430c12f5f.png)
and i don't know what does this mean
![image](https://user-images.githubusercontent.com/21698755/234822118-39e14aeb-db4e-4286-afc4-5f0ee77cfcfc.png)
if i use default text i can generate the voice, but it doesn't look like it at all, so I changed the text to Chinese and regenerated the npz file for calculation. Then an error occurred

**Comment by DrDRR** [comment]
I encountered the same AssertionError too. Still trying to figure out why. 
As for the 'text', it basically means what the audio file you provided says, I think. You should change it to match your audio file.

**Comment by africa1207** [comment]
> I encountered the same AssertionError too. Still trying to figure out why. As for the 'text', it basically means what the audio file you provided says, I think. You should change it to match your audio file.

I don't think it is the text content that needs to be translated, as there is a text_prompt variable that needs to be translated.

![image](https://user-images.githubusercontent.com/21698755/236742146-4701ace8-db85-45cf-ae92-86410c7c5d86.png)

**Comment by devinschumacher** [comment]
@africa1207 

We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉 Check it out: https://github.com/serpapps/ai-voice-cloner
